packages/client/src/pages/sign-in.tsx

```diff
@@ -99,6 +115,22 @@ export const SignIn = () => {
             <></>
           )}
         </Grid>
+        <Grid container>
+          {project?.authMethods.googleAuth ? (
+            <Grid item>
+              <GoogleLogin
+                onSuccess={({ credential }) => {
+                  loginGoogle({ variables: { projectId: project?.id || '', credential: credential as string } }).catch((err) => {
+                    console.log(err);
+                    pushMessage('Invalid Credentials');
```

Comment:
This message should be more helpful to users
```suggestion
                    pushMessage('Unable to Authenticate with Google');
```